docs/index.asciidoc

```diff
@@ -146,9 +149,11 @@ NOTE: It is an error to specify both `dictionary` and `dictionary_path`.
   * Value type is <<path,path>>
   * There is no default value for this setting.
 
-The full path of the external dictionary file. The format of the table
-should be a standard YAML, JSON, or CSV. Make sure you specify any integer-based keys
-in quotes. For example, the YAML file should look something like this:
+The full path of the external dictionary file. The format of the table should be
+a standard YAML, JSON, or CSV. Specify any integer-based keys in quotes. The
+parser is expecting a string, and the quotes make the integer-based keys
```

Comment:
Not parser. Perhaps...

> The value taken from the event via the `field` setting is always converted to a string (since version 0.1.1), therefore the lookup dictionary keys should also be strings.